app/scripts/lib/util.ts

```diff
@@ -259,31 +279,33 @@ export function formatTxMetaForRpcResult(txMeta) {
     maxPriorityFeePerGas,
   } = txParams;
 
-  const formattedTxMeta = {
+  const formattedTxMeta: FormattedTransactionMeta = {
     v,
     r,
     s,
     to,
     gas,
     from,
     hash,
-    nonce,
+    nonce: `${nonce}`,
```

Comment:
Question: if nonce is passed in as string, do we need to stringify here?

Reply:
Its likely not an issue because our response handler would have stringified it anyways but in the TxMeta type its typed as a number. I'm not sure how accurate this is but this was done as a safety precaution.

Reply:
thanks for explaining ~ it is safer this way indeed